sale_quotation_products/models/product_product.py

```diff
@@ -77,41 +83,38 @@ def write(self, vals):
             self = self.sudo()
         return super(ProductProduct, self).write(vals)
 
-    @api.one
-    def _get_qty(self):
-        self.qty = 0
+    @api.multi
+    def _compute_qty(self):
         sale_order_id = self._context.get('active_id', False)
-        if sale_order_id:
-            lines = self.env['sale.order.line'].search([
-                ('order_id', '=', sale_order_id),
-                ('product_id', '=', self.id)])
-            self.qty = sum([self.env['product.uom']._compute_qty_obj(
-                line.product_uom,
-                line.product_uom_qty,
-                self.uom_id) for line in lines])
+        for rec in self:
+            qty = 0
+            if sale_order_id:
```

Comment:
Improves of performance of this method

1.  > if sale_order_id:

    This value is the same for all the records of the loop, so is not needed to be questioned  every time,
remove from the loop and  ask for it before the loop

2. > self.env['sale.order.line'].

   define outside the loop

Example 

```python
@api.multi
def _compute_qty(self):
    sale_order_id = self._context.get('active_id', False)
    if not sale_order_id:
        self.update({'qty': 0.0})
        return

    sale_line_obj = self.env['sale.order.line']
    for rec in self:
        lines = sale_line_obj.search([
            ('order_id', '=', sale_order_id),
            ('product_id', '=', rec.id)])
        qty = sum([line.product_uom._compute_quantity(
            line.product_uom_qty,
            rec.uom_id) for line in lines])
        rec.update({'qty': qty})
```